lib/puppet/reports/http.rb

```diff
@@ -14,7 +14,7 @@
   def process
     url = URI.parse(Puppet[:reporturl])
     headers = { "Content-Type" => "application/x-yaml" }
-    options = {}
+    options = { :metric_id => [:puppet, :report, :http] }
```

Comment:
nevermind, I'm wrong. that validation is only done when creating a new connection instance, which is already done for us via the http pool